### ISSUE TITLE
Primitive.agda: openFile: does not exist (No such file or directory)

### ISSUE THREAD

**Issue by nkaretnikov** [markokoleznik/agda-writer#34]
Really excited about this project!

Compiled myself using Xcode and tried to typecheck:
```
*Error*
/Users/nikita/Library/Developer/Xcode/DerivedData/Agda_Writer-bkhyhfeszfmmsyclqnrcntveamnu/Build/Products/Debug/Agda
Writer.app/Contents/Resources/Agda-2.4.2.2/lib/prim/Agda/Primitive.agda:
openFile: does not exist (No such file or directory)
```

How do I configure it? I just ran it from the command line from the `DerivedData` directory.

**Comment by nkaretnikov** [comment]
In the terminal, I see:
```
Failed to load Agda. Reason: launch path not accessible
```

**Comment by markokoleznik** [comment]
Hello @nkaretnikov ,
I'm afraid this project is stopped due to the lack of time (and maybe motivation).
First of all, it looks like you are describing the issue mentioned here, the version of Agda above 2.5 are handling libraries a bit differently:
https://github.com/markokoleznik/agda-writer/issues/33

Other than that, if you are looking for an editor for Agda that is not emacs, I would strongly suggest checking out `agda-mode` package for Atom editor.
https://github.com/banacorn/agda-mode

Also, if you really want to push this forward, let me know. Pull Request would be even more awesome :)

**Comment by nkaretnikov** [comment]
I ended up using GUI emacs with the evil mode, which is not vim, but still better than atom (IMO).

Besides installing evil, I also configured the mappings as described in "Emulating vi inside emacs":
http://wiki.portal.chalmers.se/agda/pmwiki.php?n=Main.VIMEditing

This is not ideal, but has the advantage that all serious users seem to use emacs anyway. :\

**Comment by andrejbauer** [comment]
Perhaps @markokoleznik is willing to hand over the maintainance to anyone who would have the time?

**Comment by markokoleznik** [comment]
@andrejbauer Yes, if someone is willing to take this over, I would gladly hand over maintenance. I would even proactively help with explaining where the code is, what might be the issue and so on...

**Comment by andrejbauer** [comment]
I'll ask on the Agda mailing list.

**Comment by thosgood** [comment]
@markokoleznik  Any news on this? I don't have oodles of spare time at the moment, but would be very interested in contributing to this project with what time (and sparse knowledge) I do have.
